frontend/.env.example

```
@@ -0,0 +1,2 @@
+VITE_CLERK_PUBLISHABLE_KEY=pk_test_xxxxxxxxxxxx
+VITE_API_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_API_URL key should go before the VITE_CLERK_PUBLISHABLE_KEY key

(UnorderedKey)